content/hi/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,125 @@
+---
+title: सीग्रुप v2 के बारे में
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+Linux पर, {{< glossary_tooltip text="control groups" term_id="cgroup" >}}
+प्रक्रियाओं को आवंटित संसाधनों को सीमित करें।
+
+{{< glossary_tooltip text="क्यूबलेट" term_id="kubelet" >}} और
+अंतर्निहित कंटेनर रनटाइम को लागू करने के लिए cgroups के साथ इंटरफ़ेस करने की आवश्यकता है
+[फली और कंटेनरों के लिए संसाधन प्रबंधन](/docs/concepts/configuration/manage-resources-containers/) जो
```

Comment:
[फली और कंटेनरों के लिए संसाधन प्रबंधन] 
The word फली is not suitable here instead we can write 
`पॉड्स और कंटेनरों के लिए संसाधन प्रबंधन`